internal/resource/nvml-device.go

```diff
@@ -99,3 +101,19 @@ func (d nvmlDevice) GetPCIClass() (uint32, error) {
 	}
 	return nvDevice.Class, nil
 }
+
+func (d nvmlDevice) GetFabricIDs() (string, string, error) {
+	gfInfo, ret := d.GetGpuFabricInfo()
```

Comment:
nit: why not just:
```suggestion
	info, ret := d.GetGpuFabricInfo()
```
the `gf` prefix is not needed in this context.

Reply:
got it